week-1/Homework/mandatory/4-tax.js

```diff
@@ -5,8 +5,13 @@
   Sales tax is 20% of the price of the product
 */
 
-function calculateSalesTax() {}
-
+function calculateSalesTax(price) {
```

Comment:
Well done. However:
- Use `const` instead of `let` for tax. Tax is something that stays fixed and not suppose to change whilst the programme is running.
- Break down your code more to make it easy for other people (and yourself) to read the code. Example, create a variables called `taxRate` and make it equal to `20/100` or . If finance minster changes the rate next year, then you have on place to change it. Also, it explains to other people reading your code 20 is the tax rate.